indico/legacy/pdfinterface/conference.py

```diff
@@ -118,13 +120,24 @@ def getBody(self, story=None):
         story.append(Paragraph(u'\n<br/>\n'.join(parts).encode('utf-8'), style))
 
         story.append(Spacer(1, 0.4*inch))
-        for track in self.event.tracks:
-            bogustext = track.title.encode('utf-8')
+        tracks = [track for track in self.event.tracks if not track.track_group]
+        track_groups = self.event.track_groups
+        items = sorted(tracks + track_groups, key=lambda x: x.position)
+        for item in items:
+            bogustext = item.title.encode('utf-8')
             p = Paragraph(escape(bogustext), styles["Heading1"])
             self._story.append(p)
-            bogustext = track.description.encode('utf-8')
+            bogustext = item.description.encode('utf-8')
             p = Paragraph(escape(bogustext), style)
             story.append(p)
+            if type(item) is TrackGroup and item.tracks is not None:
```

Comment:
use `isinstance` here - and i don't think `item.tracks` can be none, can it?

Reply:
What is the value of `item.tracks` if a group has no tracks? (That was my assumption)

Reply:
should be an empty list